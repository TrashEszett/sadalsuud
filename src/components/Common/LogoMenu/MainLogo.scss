.MainLogo{
  text-align: center;
  background-color: darken(#69657f,30%);
  color: lighten(#69657f,30%);
  height: 100vh;
}
