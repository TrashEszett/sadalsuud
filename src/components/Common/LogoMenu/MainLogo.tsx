import React from 'react';

import './MainLogo.scss'

interface Props {
    userName : string;
}

const MainLogo: React.FC<Props> = ({userName}) => {
    return (
        <div className={"MainLogo"}>
            <img src={`${process.env.PUBLIC_URL}/image/Logo.png`} alt={"Main Logo - Vase and star"}/>
            <p>Welcome ! {userName}<br/>사이트 개장 준비중입니다...</p>
        </div>
    );
};

export default MainLogo;
