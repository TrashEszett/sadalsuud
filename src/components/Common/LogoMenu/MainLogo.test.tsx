import React from 'react';
import MainLogo from "./MainLogo";
import renderer from 'react-test-renderer'

test('MainLogo create', ()=>{
   const component = renderer.create(<MainLogo userName="Trㅁacy"/>);
   let tree = component.toJSON();
   expect(tree).toMatchSnapshot();
});
