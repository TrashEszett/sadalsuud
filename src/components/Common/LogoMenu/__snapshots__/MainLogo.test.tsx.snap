// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLogo create 1`] = `
<div
  className="MainLogo"
>
  <img
    alt="Main Logo - Vase and star"
    src="/image/Logo.png"
  />
  <p>
    Welcome ! 
    Tracy
    <br />
    사이트 개장 준비중입니다...
  </p>
</div>
`;
